internal/services/automation/automation_variable_int_resource_test.go

```diff
@@ -132,6 +132,7 @@ resource "azurerm_automation_variable_int" "test" {
   automation_account_name = azurerm_automation_account.test.name
   description             = "This variable is created by Terraform acceptance test."
   value                   = 12345
+  encrypted               = true
```

Comment:
as with the other PR's - since Encrypted and Unencrypted are treated as different workflows in the API, can we remove this and add a separate test to test the Encrypted lifecycle too (including that it can be updated)?

Reply:
@tombuildsstuff I've updated the PR. please bother to check again. Thanks!